src/main/java/org/madhuri/app/message/model/StockSymbol.java

```diff
@@ -0,0 +1,63 @@
+package org.madhuri.app.message.model;
+
+import jakarta.json.bind.annotation.JsonbProperty;
+
+public class StockSymbol {
+	
+	@JsonbProperty("description")
+	private String description;
+	
+	@JsonbProperty("displaySymbol")
+	private String displaySymbol;
+	
+	@JsonbProperty("symbol")
+	private String symbol;
+	
+	@JsonbProperty("type")
+	private String type;
+
+	public StockSymbol()
```

Comment:
Do we need a empty constructor ?

Reply:
The default constructor is especially used during deserialization when converting JSON or data formats into Java objects.